src/hacbs/components/topology/factories/GraphErrorState.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Alert, AlertActionCloseButton, Bullseye } from '@patternfly/react-core';
+import { OutlinedEyeSlashIcon } from '@patternfly/react-icons/dist/esm/icons/outlined-eye-slash-icon';
+import { global_palette_black_600 as grayColor } from '@patternfly/react-tokens/dist/js/global_palette_black_600';
+import cx from 'classnames';
+
+import './GraphErrorState.scss';
+
+type GraphErrorStateProps = {
+  errors: unknown[];
+  fullHeight?: boolean;
+};
+
+const GraphErrorState: React.FC<GraphErrorStateProps> = ({ errors, fullHeight }) => {
+  const [closedError, setClosedError] = React.useState<number[]>([]);
+
+  if (errors.length === 0) {
+    return null;
+  }
+  return (
+    <>
+      {errors.length !== closedError.length && (
+        <div className="hacbs-inline-errors" data-test="graph-error-state">
+          {errors.map((e: { message: string }, key) => {
+            return !closedError.includes(key) ? (
+              <Alert
```

Comment:
It's possible to get multiple alerts of the same error message.
![image](https://user-images.githubusercontent.com/14068621/206000031-542d4c3d-9cd4-4316-9657-1346523c6a49.png)

I wonder if we're better off showing a generic error message in the alert title with a collapsible details view where we show the actual returned error message(s).
Or maybe we can just de-dupe error messages.

Reply:
De-duped the error messages and added screenshots in description.